source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -102,6 +102,76 @@ class NiDCPowerDriverApiTest : public ::testing::Test {
     return response.error_message();
   }
 
+  ViBoolean get_bool_attribute(const char* channel_list, dcpower::NiDCPowerAttributes attribute_id)
+  {
+    ::grpc::ClientContext context;
+    dcpower::GetAttributeViBooleanRequest request;
+    request.mutable_vi()->set_id(GetSessionId());
+    request.set_channel_name(channel_list);
+    request.set_attribute_id(attribute_id);
+    dcpower::GetAttributeViBooleanResponse response;
+    ::grpc::Status status = GetStub()->GetAttributeViBoolean(&context, request, &response);
```

Comment:
Can you introduce new line before calling actual API to make it more readable?

Reply:
done.

Reply:
@ni-pkhare , I had asked to add new line before calling API. You can remove the line which is after calling API in all methods